test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -251,6 +277,32 @@ public void LightGbmMulticlassEstimator()
             Done();
         }
 
+        /// <summary>
+        /// LightGbmMulticlass CorrectSigmoid test 
+        /// </summary>
+        [LightGBMFact]
+        public void LightGbmMulticlassEstimatorCorrectSigmoid()
+        {
+            var (pipeline, dataView) = GetMulticlassPipeline();
+            var sigmoid = .789;
+
+            var trainer = ML.MulticlassClassification.Trainers.LightGbm(new LightGbmMulticlassTrainer.Options
+            {
+                Sigmoid = sigmoid
+            });
+
+            var pipe = pipeline.Append(trainer)
+                    .Append(new KeyToValueMappingEstimator(Env, "PredictedLabel"));
+
+            var transformedDataView = pipe.Fit(dataView).Transform(dataView);
+            var model = trainer.Fit(transformedDataView, transformedDataView);
+
+            //the slope in the all the calibrators should be equal to the negative of the sigmoid passed into the trainer
```

Comment:
nit: same suggestion regarding format of comment.

Reply:
Sounds good. I will update it.

Reply:
Fixed.